README.md

```diff
@@ -93,6 +93,9 @@ modelActions.save('posts', 42, {
   title: 'Update post'
 })
 ```
+* Note that `update` creates a new Brainstem model and will not make a request if the model is [invalid](https://github.com/mavenlink/brainstem-redux/blob/master/lib/actions/model.js#L53-L58)
+* Adding a reject handler addresses this locally
+* Note that `update` will override optional attributes, so you will need the original model's attributes
```

Comment:
Yeah, though, at some point we'll want to move towards PATCH updates. This is a good thing to keep in mind for when those efforts start rolling!